src/Contracts/Filter.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\Envy\Contracts;
+
+interface Filter
+{
+    public function environmentVariableMatches(string $environmentVariable): bool;
```

Comment:
This method name becomes rather long 🤔 
I think we should consider finding a shorter name for it 👍

Reply:
Renamed to `check`.